src/gui/qgsattributeform.cpp

```diff
@@ -223,14 +223,15 @@ void QgsAttributeForm::setMode( QgsAttributeForm::Mode mode )
   emit modeChanged( mMode );
 }
 
-void QgsAttributeForm::changeAttribute( const QString &field, const QVariant &value )
+void QgsAttributeForm::changeAttribute( const QString &field, const QVariant &value, const QString &toolTip )
 {
   Q_FOREACH ( QgsWidgetWrapper *ww, mWidgets )
   {
     QgsEditorWidgetWrapper *eww = qobject_cast<QgsEditorWidgetWrapper *>( ww );
     if ( eww && eww->field().name() == field )
     {
       eww->setValue( value );
+      eww->widget()->setToolTip( toolTip );
```

Comment:
OK so I added a `setHint` method on editor widgets. 

However I'm not sure to understand the aim of the `mBackupText` as there's also a `mPlaceholderText` attribute used for the line edit widget?